react/client/package.json

```diff
@@ -1,27 +1,36 @@
 {
-  "name": "react-opossum",
+  "name": "react",
```

Comment:
I think the naming here is confusing snyk,  maybe it should be opossum-react-client.  Up to you if you want to change the server package.json to be something similar,  but not needed

Reply:
I renamed it from **opossum-react** to **react-opossum** 😎

Reply:
i'm getting some weird npm error during the npm install:

```
npm ERR! Cannot read property 'match' of undefined
```

i removed my local node_modules folder and the package-lock.json and it worked.

Reply:
This error appeared with the last commit ?? (changing the name is package.json)

Reply:
yea,  I would just delete your node_modules folder and package-lock.json  and then do another install to recreate the package-lock.json,  at least that is what the internet said :sunglasses:

Reply:
yeah it seems to work now 👍